cmd/create/oidcprovider/cmd.go

```diff
@@ -69,6 +70,9 @@ func init() {
 	ocm.AddOptionalClusterFlag(Cmd)
 	aws.AddModeFlag(Cmd)
 
+	arguments.AddRegionFlag(flags)
+	flags.MarkHidden("region")
```

Comment:
Should we use `MarkDeprecated` instead of `MarkHidden`

Reply:
We had [this solution](https://github.com/openshift/rosa/blob/5cc4797726a3ab39f94753d76af555b22c61a030/cmd/create/cmd.go#L71) in the past which also marks hidden. I'd say we should stick to it and add the command to the list :thinking: